### PR TITLE
AV-162881 use avi certificateref in checksum calculation

### PR DESCRIPTION
**Pull request by arihantg**

### CODE REVIEW

**Comment by arihantg** [comment]
FT are added in this PR https://github.com/avinetworks/avi-dev/pull/99126

File: internal/nodes/avi_model_nodes.go [hunk 1]
```diff
@@ -166,6 +166,10 @@ func (v *AviVsNode) CalculateForGraphChecksum() uint32 {
 	for _, sslkey := range v.SSLKeyCertRefs {
 		checksumStringSlice = append(checksumStringSlice, fmt.Sprint(sslkey.GetCheckSum()))
 	}
+	for _, sslkey := range v.SSLKeyCertAviRef {
+		utils.Hash(sslkey)
```
Comment (aaha97) [inline internal/nodes/avi_model_nodes.go:170]: is this required?

Comment (arihantg) [reply internal/nodes/avi_model_nodes.go:170]: not required removing it.

**Review by akshayhavile — APPROVED** [review]

**Review by aaha97 — APPROVED** [review]
LGTM

**Comment by aaha97** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Comment by arihantg** [comment]
build ako

**Pull request merged**